samples/user-interface/haptics/src/main/res/values/strings.xml

```diff
@@ -14,25 +15,21 @@
   ~ limitations under the License.
 -->
 <resources>
-    <string name="app_name">Haptic Sampler</string>
```

Comment:
Is this now built inside the main app? How does the screens fit the rest of the samples, if it's in a single app? Shouldn't this have a haptic_samples_app_name instead, if the goal is to avoid colision?